domain-models-runtime/src/main/java/org/folio/rest/persist/ddlgen/SchemaMaker.java

```diff
@@ -323,7 +323,7 @@ private static String normalizeFieldName(String path) {
   }
 
   public static String convertDotPath2PostgresNotation(String prefix,
-    String path, boolean stringType, Index index, boolean isFullText){
+    String path, boolean stringType, Index index, boolean isFullText, boolean isFtIndex){
```

Comment:
Can you add javadoc for this method? It should make clear why isFullText and isFtIndex can be different.